store/sdk/src/test/java/org/apache/carbondata/sdk/file/ConcurrentSdkReaderTest.java

```diff
@@ -132,6 +132,45 @@ private void writeDataMultipleFiles(int numFiles, long numRowsPerFile) {
     }
   }
 
+  @Test public void testReadWithZeroBatchSize() throws InterruptedException {
```

Comment:
No need to add a concurrent test case for this scenario. 
Add a simple test case to intercept exception when reader.withBatch(0) is called

Reply:
Done!